src/hooks/search/utils.ts

```diff
@@ -311,9 +311,12 @@ export const itemInFilters = (
 };
 
 export const fetcher: Fetcher<ContentItem[], string> = queryString => {
-    return fetch(
-        (getURLPath('/api/search') as string) + '?' + queryString
-    ).then(async response => {
+    return fetch((getURLPath('/api/search') as string) + '?' + queryString, {
```

Comment:
super nitpick but you could use string template here instead of concatenation

Reply:
Yeah, it definitely needs that here.